src/Model/collections.ts

```diff
@@ -152,6 +168,19 @@ Model.prototype.getOrCreateDoc = function(collectionName, id, data) {
   return collection.getOrCreateDoc(id, data);
 };
 
+Model.prototype.getOrDefault = function<S>(subpath: Path, defaultValue: S) {
+  return this.get(subpath) ?? defaultValue as ReadonlyDeep<S>;
+};
+
+Model.prototype.getOrThrow = function<S>(subpath?: Path) {
+  const value = this.get(subpath);
+  if (value === undefined) {
```

Comment:
It'd be better to use `== null` here, which will be true when the value is `null` or `undefined`.

That matches the behavior of the `??` nullish coalescing operator used for `getOrDefault` above, and makes it usable for fields that could include `null`.